test/auro-backtotop.test.js

```diff
@@ -15,4 +15,56 @@ describe('auro-backtotop', () => {
 
     await expect(el).to.be.true;
   });
+  
+  it('tests scroll-related events', async () => {
+    window.innerHeight = 2500;
+    const el = await fixture(html`
+      <auro-backtotop cssclass="testClass"></auro-backtotop>
+    `);
+
+    simulateScroll(0, 500);
+    expect(el.hidden).to.be.false;
+
+    simulateScroll(0, 0);
+    expect(el.hidden).to.be.true;
+  });
+
+  it('tests click event', async () => {
+    const el = await fixture(html`
+      <auro-backtotop cssclass="testClass"></auro-backtotop>
+    `);
+    el.shadowRoot.querySelector('auro-button').click();
+    expect(document.documentElement.scrollTop).to.equal(0);
+  });
```

Comment:
The `expect` here only makes sense if you have scrolled down the page. You need to do what is on line 25 above before doing the click that is on 36.

Reply:
You're right, I didn't aware of that. Thanks for noticing it.